CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased Phase 3
 
+### Breaking Changes
```

Comment:
Wondering if there should be migration instructions for this... 🤔.

Reply:
Definitely requires an update to https://github.com/nasa/cumulus-api/

Reply:
Yeah, good call. I'll update that repo once all three tickets for 2688 are in as they all have changes that'll need to be noted. It looks like we might need a new `rds-phase-3` branch there that gets released at the same time as the main cumulus phase 3 release too 🤔